src/popup/components/Tokens.vue

```diff
@@ -137,7 +137,7 @@ export default defineComponent({
   }
 
   .symbol {
-    @extend %face-sans-16-medium;
+    @extend %face-sans-16-regular;
```

Comment:
Same question for this change.

Reply:
with tsvetan

Reply:
@ Odo, @CedrikNikita
We have synced about the discrepancies observed in the font display in Figma compared to font display in the browser extension and the apps. In general the fonts used in the wallet seem to look bolder than the same font used in Figma. The idea was to make the font-weights look as much as possible alike. 

However what I see following the test link is not what the expected result was supposed to be. 
These are the correct values for the widget (and for the tx items in Transactions history list): 

![image](https://user-images.githubusercontent.com/12225410/235536671-75d31f15-59ef-4254-a33e-b93fdfa4651a.png)

As improvement of the font-weight display @pstachel already suggested a change in the bold text with font-weight 700 to font weight 600 (which is part of another PR) and I approved it.